tools/hammer_loop.sh

```diff
@@ -29,6 +29,24 @@ if pgrep -fl -U "$(id -u)" "$cds"; then
     exit 1
 fi
 
+WORK_ROOT=${WORK_ROOT:-/tmp}
+TEST_ROOT="$WORK_ROOT/hammer_loop"
+REGION_ROOT=${REGION_ROOT:-/var/tmp/hammer_loop}
+if [[ ! -d "$TEST_ROOT" ]]; then
+    mkdir -p "$TEST_ROOT"
+    if [[ $? -ne 0 ]]; then
+        echo "Failed to make test root $TEST_ROOT"
+        exit 1
+    fi
+else
+    # Delete previous test data
+    rm -r "$TEST_ROOT"
```

Comment:
Should this be `rm -r "$TEST_ROOT/*"`, to avoid deleting the directory itself?

Alternatively, we could do this first if `$TEST_ROOT` exists, then **unconditionally** create the directory and check the `mkdir` return code.

Reply:
I've refactored all the test logs and region directories to always be in a sub-directory, so all this should be consistent now.